src/app/ui/timeline.h

```diff
@@ -291,6 +296,16 @@ namespace app {
 
     AniControls m_aniControls;
 
+    bool m_thumbnailsOverlayVisible;
+    gfx::Rect m_thumbnailsOverlayInner;
+    gfx::Rect m_thumbnailsOverlayOuter;
+    Hit m_thumbnailsOverlayHit;
+    gfx::Point m_thumbnailsOverlayDirection;
+    void updateCelOverlayBounds(const Hit& hit);
```

Comment:
We are using an implicit order of members, see that member variables are here, but member functions are before the variables.